app/stac_api/models.py

```diff
@@ -277,6 +280,31 @@ class Item(models.Model):
         null=False, blank=False, default=BBOX_CH, srid=4326, validators=[validate_geometry]
     )
 
+    class Meta:
+        unique_together = (('collection', 'name'),)
+        indexes = [
+            models.Index(fields=['name'], name='item_name_idx'),
+            # the following 3 indices are used e.g. in collection_temporal_extent
+            models.Index(fields=['properties_datetime'], name='item_datetime_idx'),
+            models.Index(fields=['properties_start_datetime'], name='item_start_datetime_idx'),
+            models.Index(fields=['properties_end_datetime'], name='item_end_datetime_idx'),
+            # created, updated, and title are "queriable" in the search endpoint
+            # see: views.py:322 and 323
+            models.Index(fields=['created'], name='item_created_idx'),
+            models.Index(fields=['updated'], name='item_updated_idx'),
+            models.Index(fields=['properties_title'], name='item_title_idx'),
+            # geometry is used in "filter_by_intersects", views.py:415 and filtering for bbox
+            models.Index(fields=['geometry'], name='item_geometry_idx'),
```

Comment:
not really sure about that but i think geometry index is created by default by django. i have found one in my local django installation:
```sql
CREATE INDEX stac_api_item_geometry_id
    ON public.stac_api_item USING gist
    (geometry)
    TABLESPACE pg_default;
```

https://docs.djangoproject.com/en/3.1/ref/contrib/gis/model-api/#spatial-index

Reply:
Good point @ltclm, thanks for the hint.
I don't really know much about the geometry indexing so far, most likely @rebert knows more about GeoDjango and geometric indexing.
To test it, I disabled the manually created geometry index and migrated backwards. The requests that formerly used the manually created geometry index now use an auto-generated index `stac_api_item_geometry_id`. So I guess that is fine and we can also drop the manually created index on geometry, as the auto-generated one will be fine.

Reply:
Geometry fields are automatically indexed, yes: https://docs.djangoproject.com/en/3.1/ref/contrib/gis/model-api/#spatial-index `spatial_index` is an option to every `GeometryField` which is by default `True`.